### PR TITLE
{chem}[intel/2014b] CP2K 20131211 (OK)

### PR DESCRIPTION
**Pull request by boegel**
@wpoely86: please review/test?

### CODE REVIEW

File: easybuild/easyconfigs/l/libxc/libxc-2.0.1-intel-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+sanity_check_paths = {
+    'files': ['lib/libxc.a', 'lib/libxc.so'],
+    'dirs': ['include'],
+}
+
+parallel = 1
+
+moduleclass = 'chem'
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/libxc/libxc-2.0.1-intel-2014b.eb]: needless newline?

**Comment by wpoely86** [comment]
looks fine, but should be have an easyconfig for this? I would just stick to the `v2.6.0` (or newer checkout)?

**Comment by boegel** [comment]
@wpoely86: we contribute back all easyconfigs we install in production centrally

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/585154c6de9e2a3522e5 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 4 out of 6 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/910bbe05ccd95cda7eca for a full test report.

**Comment by boegel** [comment]
@wpoely86: any idea what's going on with the failed test run?

```
configure: error: in `/home/ward/tmp/eb_sandbox_20150120_22-37-26_MdeEs/libxc/2.0.1/intel-2014b/libxc-2.0.1':
configure: error: linking to Fortran libraries from C fails
See `config.log' for more details
```

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 5 out of 6 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/6fa53ac28d66e9a3f5b2 for a full test report.

**Comment by wpoely86** [comment]
Almost there, a missing file.

The one before failed because: https://gist.github.com/d55a6fd0733cedc9293e

**Comment by wpoely86** [comment]
Test report by @wpoely86
**FAILED**
Build succeeded for 5 out of 6 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/cdd30fb92f5a4ed838d3 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/f150de3b93a4fbaa6012 for a full test report.

**Comment by boegel** [comment]
Thanks for testing/reviewing @wpoely86, going in!

**Pull request merged**